lib/index.js

```diff
@@ -1,54 +1,80 @@
-'use strict';
+"use strict";
 
-module.exports = function (context) {
-  var allowOneLiners = context.options[0] === 'allow-one-liners';
-  var maxInOneLiner = context.options[1] !== undefined ? context.options[1] : 1;
+module.exports = {
+  meta: {
+    docs: {
+      description:
+        "require `let` or `const` instead of `var` when outside of try...catch",
```

Comment:
This rule doesn't do that.

Reply:
Yeh sry, just copy pasted it from another example.